documentation/docs/02-layouts.md

```diff
@@ -97,3 +98,17 @@ If an error component has a [`load`](#loading) function, it will be called with
 ```
 
 > Server-side stack traces will be removed from `error` in production, to avoid exposing privileged information to users.
+
+```ts
```

Comment:
everywhere else in the docs, I think we put the types first and then the example code

Reply:
ahh, you're right -- i like it much better the way i wrote it, but i'll change it for consistency